runtime/compiler/CMakeLists.txt

```diff
@@ -237,11 +237,20 @@ if(OMR_OS_LINUX OR OMR_OS_OSX)
 		-Wno-deprecated
 		-Wno-enum-compare
 		-Wno-write-strings
-		-fomit-frame-pointer
 		-fasynchronous-unwind-tables
 		-Wreturn-type
 		-pthread
 	)
+	# Work-around for problem seen in x86_64-macOS builds in JDK 22
```

Comment:
There's no hyphen in "workaround"; the sentence should end with a period.

Reply:
Addressed in commit 30dc78315a57d9a6d4099349731064d4dc2bfba8.